proto/streamlit/proto/Navigation.proto

```diff
@@ -37,4 +37,6 @@ message Navigation {
     // display navigation in the sidebar
     SIDEBAR = 1;
   }
+
+  bool expanded = 5;
```

Comment:
Nit: Would probably put this field in sequence (after `page_script_hash` but before `Position`)